packages/claim_cascade_batteries/tonys/claim_cascade_config.json

```diff
@@ -0,0 +1,5 @@
+{
+  "inference_output_path": "/home/tony/utils/ClaimCascade/packages/claim_cascade/inference_input_vcs",
+  "policy_output_path": "/home/tony/utils/ClaimCascade/packages/claim_cascade/policy_input_vcs",
+  "claim_cascade_root_keys_path": "/home/tony/utils/ClaimCascade/packages/claim_cascade/root_keys"
+}
```

Comment:
if weve got 2 batteries, lets merge the common functionality and delete this one